test_runner/batch_others/test_wal_acceptor_async.py

```diff
@@ -147,6 +147,9 @@ async def run_restarts_under_load(pg: Postgres, acceptors: List[Safekeeper], n_w
     period_time = 10
     iterations = 6
 
+    test_timeout = 15 * 60  # 15 minutes
```

Comment:
* Better use `time.monotonic()`, otherwise the test may be randomly affected by clock adjustment. It _probably_ won't be big, but still.

And two nits:

* Add a comment on how this constant is chosen. E.g. 20 minutes is no good.
* Maybe replace `test_timeout` and `started_at` with `timeout_at = time.monotonic() + 15 * 60`.